templates/vue-starter-template/app/components/account/personal/DataForm.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
+const AccountType = {
+  private: "private",
+  business: "business",
+};
+
+type AccountTypeUnion = (typeof AccountType)[keyof typeof AccountType];
+
+const state = defineModel<{
+  firstName: string;
+  lastName: string;
+  salutationId: string;
+  title: string;
+  accountType: AccountTypeUnion;
+  company: string;
+  vatIds: string;
```

Comment:
why do we define it as the string? 

Maybe instead of manual definition we should use `operations["changeProfile post /account/change-profile"]["body"]`  type?